jsk_rviz_plugins/src/pie_chart_display.cpp

```diff
@@ -359,6 +363,11 @@ namespace jsk_rviz_plugins
     
   }
   
+  void PieChartDisplay::updateCaption()
+  {
+    caption_ = caption_property_->getStdString();
```

Comment:
i tried to use `getName()` but did not know where is this function and `name_` defined.